Doc/whatsnew/3.14.rst

```diff
@@ -2316,42 +2292,90 @@ gc
 * The new :ref:`incremental garbage collector <whatsnew314-incremental-gc>`
   means that maximum pause times are reduced
   by an order of magnitude or more for larger heaps.
+
+  Because of this optimisation, the meaning of the results of
```

Comment:
I guess we should follow the spelling of the section heading....

```suggestion
  Because of this optimization, the meaning of the results of
```

Reply:
Or we change the section heading!